support/mock/mockrt.go

```diff
@@ -71,14 +73,14 @@ type expectRandomness struct {
 	out abi.Randomness
 }
 
-type expectedMessage struct {
-	// expectedMessage values
+type ExpectedMessage struct {
```

Comment:
Blocking: do not export this. I think we'll be able to avoid the LastSentMessage altogether, but if we do need something like that, do not return an objet of this type: return just the bits that are needed (which I think just the return value).